dune-project

```diff
@@ -104,7 +104,7 @@ a few other projects developed at r2c.
     (yojson (>= "1.7.0"))
     (re (>= "1.10.4"))
     (pcre (>= "7.5.0" ))
-    (pcre2 (>= "7.5.3" ))
```

Comment:
@kopecs we should probably remove this no? pcre2 is "vendored" under libs/, so no
need to install it via opam no?

Reply:
Hmm, we still depend on it, so I would assume we want it present here, no?

Reply:
no, this is used to install external packages.
It's an opam thing (it's in a dune file, but dune just use it to autogenerate the semgrep.opam for OPAM).